src/Storage/StorageInterface.php

```diff
@@ -10,21 +10,37 @@
 interface StorageInterface
 {
     /**
-     * Persists counter value in the storage, uniquely referenced by a key with an optional expiration TTL time.
+     * Saves the value of key only if it was not already saved in the store.
+     * it returns false if a value was saved, Otherwise, it returns true and saves the value.
+     * the store supports expiring keys, the key will expire after the provided ttl.
```

Comment:
```suggestion
     * Saves the value of key only if it was not already saved in the store.
     * It returns `false` if a value was saved. Otherwise, it returns `true`
     * and saves the value.
     * If the storage supports expiring keys, the key will expire after the provided TTL.
```